daemon/container_unix.go

```diff
@@ -640,7 +640,7 @@ func (container *Container) updateNetworkSettings(n libnetwork.Network, ep libne
 		return err
 	}
 
-	if container.hostConfig.NetworkMode == runconfig.NetworkMode("bridge") {
+	if container.hostConfig.NetworkMode == runconfig.NetworkMode("default") {
 		networkSettings.Bridge = container.daemon.config.Bridge.Iface
```

Comment:
@tiborvass This patch will fix the a small bug of "docker inspect", which shows empty of the bridge name. But as @mavenugo said, this needs to be done in libnetwork. So I am not sure if we can apply the patch to resolve the small bug first. At the same time, I will look at the libnetwork to to integrate it with the network checkpoint/restore. (https://github.com/boucher/docker/pull/15/files)